libs/cart/src/selectors/cart.selector.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  createFeatureSelector,
+  createSelector,
+  MemoizedSelector
+} from '@ngrx/store';
+
+import {
+  State,
+  CartState
+} from '../reducers';
+
+/**
+ * Cart State
+ */
+export const selectCartState = createFeatureSelector<State>('cart');
+
+export const cartStateSelector: MemoizedSelector<object, State['cart']> = createSelector(
+  selectCartState,
+  (state: State) => state.cart
+)
+
+export const selectCartValue: MemoizedSelector<object, CartState['cart']> = createSelector(
+  cartStateSelector,
+  (state: CartState) => state.cart
+)
```

Comment:
There should also be selectors for every other major attribute of the cart model. So like

selectCartId
selectCartSubtotal
selectCartGrandTotal
selectCartCoupons
selctCartItems

and then more selectors for the rest of the cart model:
  billing_address: DaffCartAddress | null;
  shipping_address: DaffCartAddress | null;
  payment?: DaffCartPaymentMethod;
  totals: DaffCartTotal[];
  shipping_information: DaffCartShippingInformation | null;
  available_shipping_methods: DaffCartShippingRate[];
  available_payment_methods: DaffCartPaymentMethod[];

Reply:
Then these will need to be added to the cart facade too

Reply:
Changed in [`58bd413`](https://github.com/graycoreio/daffodil/pull/681/commits/58bd4135400209469b1e66d1f36c7be4f7da7c88).